src/main/groovy/jaci/openrio/gradle/wpi/toolchain/WPIRoboRioGcc.groovy

```diff
@@ -60,15 +60,6 @@ class WPIRoboRioGcc extends CrossGcc {
                     target.cppCompiler.withArguments { a ->
                         a << '--sysroot' << sysroot
                     }
-                    target.linker.withArguments { a ->
-                        a << '--sysroot' << sysroot
```

Comment:
Linker does respond to sysroot, and won't run properly without it. The linker that gradle uses is `gcc`, not `ld`